central/cluster/datastore/datastore_impl_postgres_test.go

```diff
@@ -111,7 +74,6 @@ func (s *ClusterPostgresDataStoreTestSuite) TestSearchWithPostgres() {
 	ctx := sac.WithAllAccess(context.Background())
 
 	// Upsert cluster.
-	s.netFlows.EXPECT().CreateFlowStore(gomock.Any(), gomock.Any()).Return(netFlowsMocks.NewMockFlowDataStore(s.mockCtrl), nil)
```

Comment:
how come this changed?

Reply:
because the test uses real datastores now